tests/test_diff.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+
+from sqlglot import parse_one
+from sqlglot.diff import diff, Insert, Remove, Keep
+from sqlglot.expressions import Identifier, Table
+
+
+class TestDiff(unittest.TestCase):
+    def test_simple(self):
+        self._validate_delta_only(
+            diff(parse_one("SELECT a + b"), parse_one("SELECT a - b")),
+            [
+                Remove(parse_one("a + b")),
+                Insert(parse_one("a - b")),
+            ],
+        )
+
+        self._validate_delta_only(
+            diff(parse_one("SELECT a, b, c"), parse_one("SELECT c, a")),
+            [
+                Remove(parse_one("b")),
+            ],
+        )
+
+        self._validate_delta_only(
+            diff(parse_one("SELECT a, b"), parse_one("SELECT c, b, a")),
+            [
+                Insert(parse_one("c")),
+            ],
+        )
+
+        self._validate_delta_only(
+            diff(
+                parse_one("SELECT a FROM table_one"),
+                parse_one("SELECT a FROM table_two"),
+            ),
+            [
+                Remove(
+                    Table(
```

Comment:
i have helper methods now for this to make the tests cleaner

exp.table("table_one")

Reply:
aaah, nice 👍 will use that